cloud_functions/token_refresh/streams/strava.py

```
@@ -0,0 +1,79 @@
+"""Strava specific utils."""
+import os
+import time
+from os import environ
+from typing import Any, Dict
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)